ios-base/Gemfile.lock

```diff
@@ -83,4 +83,4 @@ RUBY VERSION
    ruby 2.6.3p62
 
 BUNDLED WITH
-   1.17.2
+   2.1.4
```

Comment:
Please not use bundler at latest version 🙏 
This is the reason https://github.com/DroidKaigi/conference-app-2020/pull/447#discussion_r368418698.

Reply:
I see, thx!